client/components/ChatUserList.vue

```diff
@@ -62,16 +62,9 @@ import {computed, defineComponent, nextTick, PropType, ref} from "vue";
 import type {UserInMessage} from "../../server/models/msg";
 import type {ClientChan, ClientUser} from "../js/types";
 import Username from "./Username.vue";
+import constants from "../js/constants";
 
-const modes = {
-	"~": "owner",
-	"&": "admin",
-	"!": "admin",
-	"@": "op",
-	"%": "half-op",
-	"+": "voice",
-	"": "normal",
```

Comment:
You are dropping normal from the list, did you check that this is safe to do and it is in fact never hit?

Reply:
My mistake - normal was used on the css as a grouping name so I placed it back.
https://github.com/snowfudge/thelounge/blob/c426de9e7e18ad0393b1356575e2b0701321158f/client/css/style.css#L1809-L1811

I've also added back the "!" based on PR#354 because I don't want to break things.
Changed operator -> op as well because again, it's used on the default styling as a grouping name

https://github.com/snowfudge/thelounge/blob/c426de9e7e18ad0393b1356575e2b0701321158f/client/css/style.css#L1797-L1799